lib/index.js

```diff
@@ -17,6 +17,25 @@ var shell = require('shelljs');
 var restify = require('restify');
 
 /**
+ * checks if the path is device rooted, that is, if it has [drive letter]:\ at its beggining.
+ * @private
+ * @param {String} fpath
+ *      The file path to check
+ * @returns {Boolean}
+ *      Whether the path is rooted or not.
+ */
+function isWin32DeviceRoot(fpath) {
+    // Only for win32
+    if (process.platform !== 'win32') {
```

Comment:
I can't check this right now, my work machine doesn't have Ubuntu. I may do it while at home. (I suspect Win32, even though it's running over Linux Subsystem).